footil/time.py

```diff
@@ -0,0 +1,24 @@
+"""Helpers for time calculations."""
+
+HOUR_IN_MIN = 60
```

Comment:
Abbreviation can be interpreted as 'how many hours in minutes' when plural not used :)
```suggestion
HOUR_IN_MINS = 60
```
```suggestion
HOUR_MINS = 60
```